py/bitbox02/communication/usart/usart.py

```diff
@@ -29,8 +29,8 @@ class SerialPort(PhysicalLayer):
     (i.e. to make it implement communication.PhysicalLayer).
     """
 
-    def __init__(self, *args: Any, **kwargs: Any):
-        self.port = serial.Serial(*args, **kwargs)
+    def __init__(self, port: str, timeout: Optional[int] = None):
```

Comment:
The timeout is set for each `read` operation explicitely (see the `read` method). If we want to change it to be in the constructor (or remove it completely if you ask me...) we should change the definition of `PhysicalLayer` as well (note that this has been defined to be compatible with the `hid` class, so there's no reason not to adapt both to our needs).

Reply:
fixed